controllers/users.js

```diff
@@ -39,6 +40,14 @@ const UsersController = {
   },
 
   Create: (req, res) => {
+    const errors = validationResult(req);
+
+    if (!errors.isEmpty()) {
+      return res.status(422).render("users/new", {
+        message: JSON.stringify(errors.array()[0].msg).replaceAll('"', ""),
```

Comment:
This line can be replaced with `message: errors.array()[0].msg,` (tested manually and all tests with it and all tests still pass)